annotation/tests/test_distribution.py

```diff
@@ -1856,7 +1856,8 @@ def test_distribute_extensive_coverage(
         ([], JobStatusEnumSchema.finished),
     ),
 )
-def test_redistribute(
+@pytest.mark.asyncio
```

Comment:
Remove this decorator from all tests and add configuration to pyproject.toml for pytest to automatically include it
https://pytest-asyncio.readthedocs.io/en/latest/reference/configuration.html